macos/generate_wazuh_packages.sh

```diff
@@ -32,7 +32,7 @@ NOTARIZE="no"                         # Notarize the package for macOS Catalina.
 DEVELOPER_ID=""                       # Apple Developer ID.
 ALTOOL_PASS=""                        # Temporary Application password for altool.
 pkg_name=""
-
+set -x
```

Comment:
I have also a question, why did you remove the quotes from line 146?

Reply:
With the quotes bash takes it as a string and will check if the version string contains the substring `^2\.` instead of checking if it starts with `2` followed by `.`, other option for letting the quotes would be:
```
if [[ "${VERSION}" =~ ^"2." ]]; then 
```

`set -x` removed in https://github.com/wazuh/wazuh-packages/pull/348/commits/0bce64bb52521741afb5398ddb2a6a374ef65159

Reply:
Cool! Thanks :)